charts/kong/values.yaml

```
@@ -19,6 +19,12 @@ deployment:
     # Setting this to false with ingressController.enabled=true will create a
     # controller-only release.
     enabled: true
+    # Deploy as daemon set
+  ## Setting this to true will deploy kong as a daemonset
+  ## Else by default it creates as a deployment
+  ##
+  # daemonset: true
+  ##
```

Indentation here is a bit wonky because of the `kong` block above. This applies to the entire replication controller, so it should be only one level below `deployment`

Also making the setting always defined with its default explicitly set. Leaving it undefined by default works since that makes it effectively `false`, but I'd generally prefer to state that explicitly:
```suggestion
  # Deploy as a DaemonSet. By default, the chart creates a Deployment.
  # Toggling this setting to true will use a DaemonSet instead
  daemonset: false
```